edivorce/apps/core/templates/question/01_orders.html

```diff
@@ -216,85 +216,107 @@ <h1>
-            and the
-            <a href="https://www2.gov.bc.ca/gov/content/life-events/divorce/family-justice/your-options/going-to-court/which-court-should-i-go-to" target="_blank">Government of B.C. website</a>.
-        </p>
-        <p><strong>Asking for Orders Pertaining to Children in Provincial or Supreme Court.</strong></p>
-        <p>
-            If you will be asking for parenting and support issues, you can choose between the
-            <a href="http://wiki.clicklaw.bc.ca/index.php?title=Divorce Act" target="_blank">Divorce Act</a>
-            or the
-            <a href="https://www2.gov.bc.ca/gov/content/justice/about-bcs-justice-system/legislation-policy/legislation-updates/family-law-act"
-               target="_blank">Family Law Act</a>.
-            If you choose to go with
-            the Family Law Act, you can get court orders or enforce an agreement in
-            either
-            <span class="tooltip-link"
-              data-toggle="tooltip" data-placement="right" data-html="true"
-              title="
-                <b>Provincial Court</b>
-                <p>The lower level trial court which deals with small claims, family,
-                youth criminal offenders, criminal and traffic matters.</p>
-                ">
-                Provincial Court<i class="fa fa-question-circle" aria-hidden="true"></i></span>
-            or
-            <span class="tooltip-link"
-              data-toggle="tooltip" data-placement="right" data-html="true"
-              title="
-                <b>Supreme Court</b>
-                <p>The higher level trial court. This court also hears appeals from
-                the Provincial Court, and sometimes reviews the decisions of certain
-                provincial tribunals and public decision-making bodies.</p>
-                ">
-            Supreme Court<i class="fa fa-question-circle" aria-hidden="true"></i></span>,
-            but if you go with the Divorce Act, you can only do this in Supreme Court.
-        </p>
-        <p>
-            Please note if you are <strong>asking for a divorce</strong>,  divorces are granted
-            under the federal Divorce Act. If you are seeking a divorce, you can
-            apply to the Supreme Court to settle your other family law issues,
-            such as support and
-            <a href="http://www2.gov.bc.ca/gov/content/life-events/divorce/family-justice/family-law/parenting-apart"
-               target="_blank">parenting arrangements</a> , under the Divorce Act or
-            under the provincial Family Law Act. The Family Law Act is usually
-            better for settling these issues because it is tailored to respond
-            to the needs of families.
-        </p>        
-    </div>
+                <div class="collapse-trigger collapsed" data-toggle="collapse"
+                aria-expanded="false" data-target="#collapseTalk"
+                aria-controls="collapseTalk">
+                    <div>
```

Comment:
Removed the numbered list, as that was inconsistent with other expandable sections.
![Screen Shot 2020-02-18 at 3 32 07 PM](https://user-images.githubusercontent.com/38964451/74787637-80d6f600-5264-11ea-8a30-4af6127ac807.png)